### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by franksouza183** [cognitive-engineering-lab/mdbook-quiz#23]
Is there any way to configure the language and customize the terms at runtime?

**Comment by willcrichton** [comment]
No, but I would be happy to accept a contribution to permit such configurations.
